### PR TITLE
Added a discord bot that returns ascii art with a input of a image link

### PR DESCRIPTION
**Pull request by crebro**
I copied some code from the "Kite Auto Complete" Youtube channel and basically converted it into a discord bot.

### CODE REVIEW

**Comment by crebro** [comment]
There was already a folder called discord in the python-programs folder, so I thought that was where the discord bots should go. Also, there was another discord bot in the same folder. sooo, do you still want me to create another folder for the bot?

File: Python-Programs/Discord/image-to-ascii/main.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import discord
+import urllib.request
+import string
+import random
+from PIL import Image
+import os
+
+client = discord.Client()
+token = "TOEKN GOES HERE"
```
Comment (thenishantsapkota) [inline Python-Programs/Discord/image-to-ascii/main.py:9]: Not wise to specify the token in the actual program.
Use `dotenv` to fetch the Bot Token from `.env` file.

**Comment by crebro** [comment]
yes, I thought of doing that, I will do that and create another pr, do you want me to create another folder for it doe

**Comment by thenishantsapkota** [comment]
> yes, I thought of doing that, I will do that and create another pr, do you want me to create another folder for it doe

Yes Please create another folder for your project inside `Hacktoberfest-2021/Python-Programs`

**Comment by crebro** [comment]
yea sure, ya got it

**Comment by crebro** [comment]
Done nishant dai

**Comment by thenishantsapkota** [comment]
Great!
I am merging this PR now!
Happy Hacktoberfest

**Pull request merged**